assets/js/googlesitekit/datastore/site/connection.js

```diff
@@ -161,10 +163,12 @@ export const selectors = {
 	 * ```
 	 * {
 	 *   connected: <Boolean>,
-	 *   resettable: <Boolean,
+	 *   resettable: <Boolean>,
+	 *   setupCompleted: <Boolean>,
 	 * }
 	 * ```
 	 *
+	 * @private
```

Comment:
I've marked this selector as private going-forward, as we expose more specific selectors for these values. We obviously don't want to make a habit of quickly deprecating selectors, but this is fine for now I think 😄